$NetBSD: patch-ac,v 1.1 2007/07/25 02:44:10 markd Exp $

Non GNU sed's don't like \b and its not really needed.

--- konversation/src/konversationircprotocolhandler.orig	2007-07-24 16:42:17.000000000 +1200
+++ konversation/src/konversationircprotocolhandler
@@ -4,7 +4,7 @@
 # A script for Konversation to call connectToServer for each instance of Konversation.
 
 url=`echo $1 | sed -e "s,irc://,,"`
-server_and_port=`echo $url | sed -e "s,\b/.*,,"`
+server_and_port=`echo $url | sed -e "s,/.*,,"`
 server=`echo $server_and_port| awk -F":" '{print $1}'`
 port=`echo $server_and_port| awk -F":" '{print $2}'`
 channel_and_password=`echo $url | awk -F"/" '{print $2}'`
